styles/gemini/catppuccin.user.css

```diff
@@ -0,0 +1,162 @@
+@description Soothing pastel theme for Gemini
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@-moz-document domain('gemini.google.com') {
+  body.dark-theme {
+    #catppuccin(@darkFlavor, @accentColor);
+  }
+
+  body.light-theme {
+    #catppuccin(@lightFlavor, @accentColor);
+  }
+
+  #rgbify(@color) {
+    @rgb-raw: red(@color), green(@color), blue(@color);
+  }
+
+  #catppuccin(@lookup, @accent) {
+    @rosewater: @catppuccin[@@lookup][@rosewater];
+    @flamingo: @catppuccin[@@lookup][@flamingo];
+    @pink: @catppuccin[@@lookup][@pink];
+    @mauve: @catppuccin[@@lookup][@mauve];
+    @red: @catppuccin[@@lookup][@red];
+    @maroon: @catppuccin[@@lookup][@maroon];
+    @peach: @catppuccin[@@lookup][@peach];
+    @yellow: @catppuccin[@@lookup][@yellow];
+    @green: @catppuccin[@@lookup][@green];
+    @teal: @catppuccin[@@lookup][@teal];
+    @sky: @catppuccin[@@lookup][@sky];
+    @sapphire: @catppuccin[@@lookup][@sapphire];
+    @blue: @catppuccin[@@lookup][@blue];
+    @lavender: @catppuccin[@@lookup][@lavender];
+    @text: @catppuccin[@@lookup][@text];
+    @subtext1: @catppuccin[@@lookup][@subtext1];
+    @subtext0: @catppuccin[@@lookup][@subtext0];
+    @overlay2: @catppuccin[@@lookup][@overlay2];
+    @overlay1: @catppuccin[@@lookup][@overlay1];
+    @overlay0: @catppuccin[@@lookup][@overlay0];
+    @surface2: @catppuccin[@@lookup][@surface2];
+    @surface1: @catppuccin[@@lookup][@surface1];
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+
+    color-scheme: if(@lookup =latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+
+    --bard-color-primary: @accent-color;
+    --bard-color-on-primary-container: @text;
+    --bard-color-surface-container: @base;
+    --bard-color-surface-container-low: lighten(@mantle, 2%);
+    --bard-color-surface-container-highest: lighten(@base, 5%);
+    --bard-color-surface-container-high: @surface0;
+    --bard-color-inverse-on-surface: darken(@mantle, 10%);
+    --bard-color-outline-variant: @subtext0;
+    --bard-color-main-container-background: @mantle;
+    --bard-color-main-container-background-rgb: #rgbify(@mantle) [];
```

Comment:
```suggestion
    --bard-color-main-container-background: @base;
    --bard-color-main-container-background-rgb: #rgbify(@base) [];
```

Reply:
Are you sure? these variables controls chat container color. In original gemini theme the left panel is lighter where the chat container is darker, nextjs theme also has `@mantle` as primary background, and `@base` for navbar 

original
![image](https://github.com/catppuccin/userstyles/assets/91272905/9459fbff-c9d7-458a-8948-523b55d736b1)

catppuccin currently:
![image](https://github.com/catppuccin/userstyles/assets/91272905/d3c62014-3bb1-4b14-aeef-69d7d4081157)


catppuccin(with your changes)
![image](https://github.com/catppuccin/userstyles/assets/91272905/e0b4985a-e4eb-4f63-bc6b-31f8f5907e56)

Reply:
Well yeah you'll want to change the sidebar colors to mantle. I'm not sure why the Next.js userstyle has it that way - that is a mistake! The *base* / primary background should always be base.